websites/G/Glimesh/dist/metadata.json

```diff
@@ -0,0 +1,54 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "service": "Glimesh",
+    "author": {
+      "name": "Nathaniel.",
+      "id": "147891648628654082"
+    },
+    "contributors": [
+        {
+            "name": "zjtech123",
+            "id":  "261236127581601793"
+        }
+    ],
```

Comment:
![image](https://user-images.githubusercontent.com/9348108/113905951-ffc61f80-97d3-11eb-83b0-2edf9319406a.png)

Reply:
Requested Changes, fixed in newest commit! :)